src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -67,9 +70,9 @@ public function authorizePayment(OrderTransfer $orderTransfer);
      *
      * @param int $idPayment
      *
-     * @return \SprykerEco\Zed\Payone\Business\Api\Response\Container\AuthorizationResponseContainer
+     * @return \Generated\Shared\Transfer\AuthorizationResponseTransfer
      */
-    public function preAuthorizePayment($idPayment);
+    public function preAuthorizePayment(int $idSalesOrder): AuthorizationResponseTransfer;
```

Comment:
* Adding a type hints to an existing API method it's a major.
* But you can fix the document block, that's okay.

Reply:
I did this chances with thoughts that it will be major (due to DB changes)
I'll check and rollback all such changes